bundle/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -101,16 +102,44 @@ spec:
       description: Nexus IQ Server
       displayName: NexusIQ
   description: |-
-    Nexus IQ Server is a policy engine powered by precise intelligence on open source components.
+    Nexus Lifecycle is an open source governance platform that continuously identifies risk,
+    automatically enforces policy, and provides visibility throughout the entire SDLC.
+    Developers leveraging open source require a solution that helps them make safer choices
+    while still delivering at DevOps speed.
```

Comment:
Why is this description different than the one on lines 86-87?

Reply:
the earlier description is just a shorter summary that shows in the smaller list views. this description is the big one that shows when you're about to install (and after it's installed.). That short description above is even a bit long, as it seems to truncate it a little in some displays.

Reply:
Ah. A little confusing since they both have the same key :disappointed:

Minor: a quick yaml comment might make it easier for future maintainers

Reply:
ah, yaml's hard to read and navigate. they're at different paths: "spec.description" vs "metadata.annotations.description" (I believe.)